bin/impl/install/accumulo.sh

```diff
@@ -18,6 +18,13 @@
 # shellcheck source=bin/impl/util.sh
 source "$UNO_HOME"/bin/impl/util.sh
 
+function install_test_jar() {
+  local test_jar_source="$DOWNLOADS/accumulo-test-2.1.0-SNAPSHOT.jar"
+  local test_jar_destination="$ACCUMULO_HOME/lib"
+  print_to_console "Installing Apache Accumulo test jar $test_jar_source to $test_jar_destination"
+  mv "$test_jar_source" "$test_jar_destination"
```

Comment:
Should this be `cp` ?
```suggestion
  cp "$test_jar_source" "$test_jar_destination"
```

Reply:
We do a `tar xzf "$DOWNLOADS/$ACCUMULO_TARBALL" -C "$INSTALL"`